frontend/packages/ceph-storage-plugin/src/types.ts

```diff
@@ -246,17 +246,20 @@ export const VaultAuthMethodMapping: {
   [keys in VaultAuthMethods]: {
     name: string;
     value: VaultAuthMethods;
+    isWizardFlowOnly: boolean;
     supportedEncryptionType: KmsEncryptionLevel[];
   };
 } = {
   [VaultAuthMethods.KUBERNETES]: {
     name: 'Kubernetes',
     value: VaultAuthMethods.KUBERNETES,
+    isWizardFlowOnly: true,
     supportedEncryptionType: [KmsEncryptionLevel.CLUSTER_WIDE, KmsEncryptionLevel.STORAGE_CLASS],
```

Comment:
don't we already have `supportedEncryptionType` list to enable/disable this feature from UI. Why do we need `isWizardFlowOnly` ?

Reply:
we just don't want to create a SC, do we really need this `isWizardFlowOnly` for that ?

Reply:
We need to display this option only for the install wizard flow and not on the storage class creation page. supportedEncryptionType will disable on both places.

Requriment is in wizard flow I have to display "kubernetes" auth method option under vault even when storage class encryption is clicked but I should not create an encrypted storage class. Also, i should not display  "kubernetes"  auth method option under vault on storage class creation screen.

Reply:
got it... never mind...